grafana/provisioning/datasources/datasource.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: 1
```

Comment:
Feedback from the review:

- Look into if it is possible to show pending requests. Thus requests that are still in flight and need to be processed.
- Look into if the titles are consistent with the type. E.g. if the graph has a type which can change (us, ms, seconds), do not mention it in the title or table header.

Reply:
A note regarding _pending requests_: this seems to be almost the same as _Number of active connections_ in the _Database Performance_ dashboard, but not quite. For example, a given client call to a gRPC method (= a "request") could involve multiple database queries. Moreover, the number of ongoing (incomplete) query executions and the number of active connections isn't necessarily a one-to-one relationship (depending on the definition of "active" I guess).

Reply:
I don't think the number of active connections are very important in gRPC. All the clients connected to the gRPC server should reuse their connection as long as possible and only re-establish it if it drops, which is 1 per API process. The number rpc calls in flight is a lot more useful.

Reply:
Looking into the metric again. I gave it a poor name. I renamed it to in flight requests. The metric represents the unary rpc requests which are in flight.

I will no longer implement the number of active connections as Amund is right, those aren´t very important.